saskatoon/sitebase/views.py

```diff
@@ -63,6 +64,12 @@ def get_context_data(self, **kwargs):
 
         return context
 
+    def post(self, *args, **kwargs):
+        self.request.user.agreed_terms = True
+        self.request.user.save()
+        return redirect('home')
```

Comment:
this feels like a shoddy workaround, but going with a form for a single field felt like overkill.

not sure how to go about this 😖

Reply:
that's totally fine!!